crates/rooch/src/commands/account/commands/sign.rs

```diff
@@ -5,57 +5,66 @@ use crate::cli_types::{CommandAction, WalletContextOptions};
 use async_trait::async_trait;
 use clap::Parser;
 use hex::ToHex;
+use moveos_types::state::MoveState;
 use rooch_key::keystore::account_keystore::AccountKeystore;
-use rooch_rpc_api::jsonrpc_types::account_sign_view::AccountSignView;
 use rooch_types::{
-    address::ParsedAddress,
-    error::{RoochError, RoochResult},
+    address::BitcoinAddress,
+    error::RoochResult,
+    framework::auth_payload::{AuthPayload, SignData, MESSAGE_INFO, MESSAGE_INFO_PREFIX},
 };
 
-/// Sign an msg with current account private key (sign_hashed)
-///
-/// This operation must be specified with -a or
-/// --address to export only one address with a private key.
+/// Sign a message with the bitcoin address
 #[derive(Debug, Parser)]
 pub struct SignCommand {
-    #[clap(short = 'a', long = "address", value_parser=ParsedAddress::parse, default_value = "")]
-    address: ParsedAddress,
+    // the address to be used
+    #[clap(short = 'a', long = "btc-address")]
+    bitcoin_address: BitcoinAddress,
```

Comment:
Use the `ParsedAddress` to support all address

Reply:
> Use the `ParsedAddress` to support all address

Used.